jobs/base_model_validator.py

```diff
@@ -185,3 +205,39 @@ def _get_model_id_regex(self):
             str. A regex pattern to be followed by the model id.
         """
         return '^[A-Za-z0-9-_]{1,%s}$' % base_models.ID_LENGTH
+
+
+class BaseSnapshotMetadataModelValidator(beam.PTransform):
```

Comment:
In this validator we should also verify the stuff that is right now verified in the `BaseModelValidator`, similar how it worked in the previous implementation when all the base validators inherit from the main base validator.

Reply:
Done.